obs-studio-client/source/nodeobs_settings.cpp

```diff
@@ -447,6 +447,69 @@ Napi::Value settings::OBS_settings_getListCategories(const Napi::CallbackInfo& i
 	return categories;
 }
 
+Napi::Array devices_to_js(const Napi::CallbackInfo& info, std::vector<ipc::value> response)
+{
+	Napi::Array devices = Napi::Array::New(info.Env());
+
+	uint32_t js_array_index = 0;
+	uint64_t items = response[1].value_union.ui64;
```

Comment:
maybe it's better to send `response` via const ref?

Reply:
good point / good catch, making the change now.